src/xcode/ENA/ENA/Source/Services/CoronaTestService/__tests__/CoronaTestServiceTests.swift

```diff
@@ -3169,29 +3200,29 @@ class CoronaTestServiceTests: CWATestCase {
 						return
 					}
 					expectation.fulfill()
-
+					
 					XCTAssertFalse(resource.locator.isFake)
 					XCTAssertEqual(count, 0)
-
+					
 					count += 1
 				}
-			)
+			),
+			LoadResource(
+				result: .success(
+					SubmissionTANModel(submissionTAN: "fake")
+				),
+				willLoadResource: { _ in
+					expectation.fulfill()
+					XCTAssertEqual(count, 1)
```

Comment:
I think at this place it should also count up `count += 1`.
The test is testing the "playbook". If I remember "playbook" means the combination of real and fake requests.